container/build-image/Dockerfile

```diff
@@ -30,6 +32,11 @@ RUN apt-get update \
 		openssl \
 		libengine-pkcs11-openssl
 
+COPY gardenlinux.asc $GARDENLINUX_MIRROR_KEY
+RUN echo "deb [signed-by=$GARDENLINUX_MIRROR_KEY] https://repo.gardenlinux.io/gardenlinux $VERSION main" >> /etc/apt/sources.list \
```

Comment:
I know you may use multiple `run` commands for readability when it comes to complex commands but (IMHO) I think we may first copy the file and combine afterwards all runs (but **not** all within a single `apt-get` call) in a single `run` command like in https://github.com/gardenlinux/gardenlinux/blob/main/container/integration-test/Dockerfile. Indention and comments may help here for further readability. What do you think?

Reply:
Good point. I think we should streamline this and make sure to create as few layers as possible. Therefore, I adjusted the Dockerfile.